src/bookshelf/src/bksw_wm_sql_object.erl

```diff
@@ -0,0 +1,315 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
```

Comment:
This file has a lot of TODO comments.  We should do them or excise the comments.